package.json

```diff
@@ -1,10 +1,18 @@
 {
-  "name": "halacious",
-  "version": "5.0.1",
+  "name": "@modernpoacher/halacious",
```

Comment:
Should keep the original name here. Ps. I hope this will get merged some day. Great work!

Reply:
Ah, yes -- that's so it works in _my_ project!

Reply:
Closing in favour of an [amended PR](https://github.com/bleupen/halacious/pull/113)